ports/tomlplusplus/vcpkg.json

```
@@ -1,7 +1,8 @@
 {
   "name": "tomlplusplus",
   "version": "2.5.0",
+  "port-version": 1,
   "description": "Header-only TOML config file parser and serializer for modern C++.",
   "homepage": "https://marzer.github.io/tomlplusplus/",
-  "supports": "!(arm | uwp | osx)"
+  "supports": "!(arm | uwp)"
```

Can you also remove the arm restriction? At least on arm64-osx it is working. 
```suggestion
  "supports": "!uwp"
```